src/main/java/essenmakanan/recipe/RecipeList.java

```diff
@@ -3,8 +3,23 @@
 import java.util.ArrayList;
 
 public class RecipeList extends ArrayList<Recipe> {
```

Comment:
Is there a need to extend the ArrayList<Recipe>? This is because we are already initialising recipes as an ArrayList and ArrayList itself has functions that we can use. Is the extending part for clarity purposes? Should we remove it or leave it?